app/controllers/users_controller.rb

```diff
@@ -365,4 +366,20 @@ def acknowledge_cookies
     end
     true
   end
+
+  def admin_search
```

Comment:
this should have a authorization check right?

Reply:
Oh yes, right. I've added now. Thanks.